custom-vu/src/main/kotlin/jces1209/vu/page/dashboard/dc/DcDashboardPage.kt

```diff
@@ -0,0 +1,76 @@
+package jces1209.vu.page.dashboard.dc
+
+import com.atlassian.performance.tools.jiraactions.api.WebJira
+import com.atlassian.performance.tools.jiraactions.api.page.wait
+import jces1209.vu.page.JiraCloudProjectList
+import jces1209.vu.page.dashboard.DashboardPage
+import org.openqa.selenium.By
+import org.openqa.selenium.support.ui.ExpectedConditions
+import java.time.Duration
+
+class DcDashboardPage(
+    private val jira: WebJira
+) : DashboardPage(
+    driver = jira.driver,
+    uri = jira.base.resolve("/secure/ConfigurePortalPages!default.jspa")) {
```

Comment:
What about to change to popular? it contains more items than favorites
```suggestion
    uri = jira.base.resolve("/secure/ConfigurePortalPages.jspa?view=popular")) {
```